src/main/java/org/skyscreamer/jsonassert/LocationAwareValueMatcher.java

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2012 Duncan Mackinder.
```

Comment:
There may be contributions from many people over time, and I don't want to track them in per-file license agreements. Can you remove this line from the files in your PR?